### ISSUE TITLE
Getting "connection was reset" when trying to open photoprism

### ISSUE THREAD

**Issue by SinTan1729** [photoprism/photoprism#1773]
### Discussed in https://github.com/photoprism/photoprism/discussions/1772

<div type='discussions-op-text'>

<sup>Originally posted by **SinTan1729** November 28, 2021</sup>
I'm running the latest update of PhotoPrism (28/11).
Screenshot:
![image](https://user-images.githubusercontent.com/35728385/143808423-44364723-4313-406f-bfeb-97549826318e.png)

Here's the output of `docker-dompose logs --tail=25`

```
photoprism-mariadb-1  | 2021-11-29 02:34:07+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
photoprism-mariadb-1  | 2021-11-29 02:34:07+00:00 [Note] [Entrypoint]: Entrypoint script for MariaDB Server 1:10.6.5+maria~focal started.
photoprism-mariadb-1  | 2021-11-29  2:34:07 0 [Note] mysqld (server 10.6.5-MariaDB-1:10.6.5+maria~focal) starting as process 1 ...
photoprism-mariadb-1  | 2021-11-29  2:34:07 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
photoprism-mariadb-1  | 2021-11-29  2:34:07 0 [Note] InnoDB: Number of pools: 1
photoprism-mariadb-1  | 2021-11-29  2:34:07 0 [Note] InnoDB: Using crc32 + pclmulqdq instructions
photoprism-photoprism-1  | started as root
photoprism-photoprism-1  | running as root
photoprism-photoprism-1  | photoprism start
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Note] InnoDB: Using Linux native AIO
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Note] InnoDB: Initializing buffer pool, total size = 134217728, chunk size = 134217728
photoprism-photoprism-1  | time="2021-11-29T02:34:13Z" level=info msg="Help us make a difference and become a sponsor today!"
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Note] InnoDB: Completed initialization of buffer pool
photoprism-photoprism-1  | time="2021-11-29T02:34:13Z" level=info msg="Visit https://docs.photoprism.org/funding/ to learn more."
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Note] InnoDB: 128 rollback segments are active.
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Note] InnoDB: Creating shared tablespace for temporary tables
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Note] InnoDB: Setting file './ibtmp1' size to 12 MB. Physically writing the file full; Please wait ...
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Note] InnoDB: File './ibtmp1' size is now 12 MB.
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Note] InnoDB: 10.6.5 started; log sequence number 1859813833; transaction id 6823241
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Note] Plugin 'FEEDBACK' is disabled.
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Warning] You need to use --log-bin to make --expire-logs-days or --binlog-expire-logs-seconds work.
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Note] Server socket created on IP: '0.0.0.0'.
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Note] Server socket created on IP: '::'.
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Warning] 'proxies_priv' entry '@% root@37f47200da29' ignored in --skip-name-resolve mode.
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [ERROR] Incorrect definition of table mysql.event: expected column 'definer' at position 3 to have type varchar(, found type char(141).
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [ERROR] mysqld: Event Scheduler: An error occurred when initializing system tables. Disabling the Event Scheduler.
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Note] mysqld: ready for connections.
photoprism-mariadb-1     | Version: '10.6.5-MariaDB-1:10.6.5+maria~focal'  socket: '/run/mysqld/mysqld.sock'  port: 3306  mariadb.org binary distribution
photoprism-mariadb-1     | 2021-11-29  2:34:08 0 [Note] InnoDB: Buffer pool(s) load completed at 211129  2:34:08
```
System info :
```
inxi -b

System:    Host: sintan-laptop Kernel: 5.10.82-1-lts x86_64 bits: 64 Desktop: KDE Plasma 5.23.3
           Distro: EndeavourOS
Machine:   Type: Laptop System: HP product: HP Pavilion Laptop 14-bf1xx v: Type1ProductConfigId
           serial: <superuser required>
           Mobo: HP model: 83CE v: 59.38 serial: <superuser required> UEFI: Insyde v: F.31
           date: 10/30/2017
Battery:   ID-1: BAT1 charge: 39.4 Wh (93.1%) condition: 42.3/42.3 Wh (100.0%)
CPU:       Info: Quad Core Intel Core i7-8550U [MT MCP] speed: 800 MHz min/max: 400/4000 MHz
Graphics:  Device-1: Intel UHD Graphics 620 driver: i915 v: kernel
           Device-2: NVIDIA GM108M [GeForce 940MX] driver: nouveau v: kernel
           Device-3: Chicony HP Wide Vision HD Camera type: USB driver: uvcvideo
           Display: x11 server: X.org 1.21.1.1 driver: loaded: intel,modesetting
           unloaded: fbdev,vesa resolution: <missing: xdpyinfo>
           Message: Unable to show advanced data. Required tool glxinfo missing.
Network:   Device-1: Realtek RTL8111/8168/8411 PCI Express Gigabit Ethernet driver: r8169
           Device-2: Intel Wireless 7265 driver: iwlwifi
           Device-3: Intel Bluetooth wireless interface type: USB driver: btusb
Drives:    Local Storage: total: 1.36 TiB used: 786.27 GiB (56.3%)
Info:      Processes: 276 Uptime: 2h 32m Memory: 15.55 GiB used: 3.98 GiB (25.6%) Shell: fish
           inxi: 3.3.09
```</div>

**Comment by SinTan1729** [comment]
Solved by doing `docker-compose down`,`docker image prune -a`, restarting and then `docker-compose up -d`.

**Comment by SinTan1729** [comment]
It's back again after reboot.

**Comment by SinTan1729** [comment]
Here's some output of `docker-compose logs photoprism-photoprism-1 --tail=100` (there's a line with `fatal` so that might be the issue) :
```
time="2021-11-29T06:11:33Z" level=info msg="metadata: updated 45 photos"
time="2021-11-29T06:11:35Z" level=info msg="index: updating counts"
time="2021-11-29T06:11:35Z" level=info msg="index: updating covers"
time="2021-11-29T07:22:28Z" level=info msg="photos: found 60 results for public:true dist:20 year:0 month:0 quality:3 count:60 order:newest merged:true [396.449022ms]"
time="2021-11-29T07:22:31Z" level=info msg="photos: found 60 results for public:true dist:20 year:0 month:0 quality:3 count:60 offset:60 order:newest merged:true [463.6247ms]"
time="2021-11-29T07:22:31Z" level=info msg="photos: found 360 results for public:true dist:20 year:0 month:0 quality:3 count:360 order:newest merged:true [500.209512ms]"
time="2021-11-29T07:24:42Z" level=info msg="photos: found 60 results for public:true dist:20 year:0 month:0 quality:3 count:60 order:newest merged:true [430.819152ms]"
time="2021-11-29T07:24:45Z" level=info msg="photos: found 360 results for public:true dist:20 year:0 month:0 quality:3 count:360 order:newest merged:true [472.414291ms]"
time="2021-11-29T07:24:45Z" level=info msg="photos: found 60 results for public:true dist:20 year:0 month:0 quality:3 count:60 offset:60 order:newest merged:true [475.714178ms]"
time="2021-11-29T07:24:50Z" level=info msg="photos: found 60 results for q:rupam&me public:true dist:20 year:0 month:0 quality:3 count:60 order:newest merged:true [45.876861ms]"
time="2021-11-29T07:24:52Z" level=info msg="photos: found 60 results for q:rupam&me public:true dist:20 year:0 month:0 quality:3 count:60 offset:60 order:newest merged:true [32.709716ms]"
time="2021-11-29T07:24:52Z" level=info msg="photos: found 360 results for q:rupam&me public:true dist:20 year:0 month:0 quality:3 count:360 order:newest merged:true [108.422132ms]"
time="2021-11-29T07:25:02Z" level=info msg="photos: found 60 results for q:rupam&me public:true dist:20 year:0 month:0 quality:3 count:60 offset:120 order:newest merged:true [25.60882ms]"
time="2021-11-29T07:25:08Z" level=info msg="photos: found 60 results for q:rupam&me public:true dist:20 year:0 month:0 quality:3 count:60 offset:180 order:newest merged:true [24.988295ms]"
time="2021-11-29T07:25:13Z" level=info msg="photos: found 60 results for q:rupam&me public:true dist:20 year:0 month:0 quality:3 count:60 offset:240 order:newest merged:true [27.229937ms]"
time="2021-11-29T07:25:19Z" level=info msg="photos: found 60 results for q:rupam&me public:true dist:20 year:0 month:0 quality:3 count:60 offset:300 order:newest merged:true [25.667606ms]"
time="2021-11-29T07:25:24Z" level=info msg="photos: found 60 results for q:rupam&me public:true dist:20 year:0 month:0 quality:3 count:60 offset:360 order:newest merged:true [26.374095ms]"
time="2021-11-29T07:25:34Z" level=info msg="photos: found 60 results for q:rupam&me public:true dist:20 year:0 month:0 quality:3 count:60 offset:420 order:newest merged:true [24.566052ms]"
time="2021-11-29T07:25:42Z" level=info msg="photos: found 60 results for q:rupam&me public:true dist:20 year:0 month:0 quality:3 count:60 offset:480 order:newest merged:true [28.158265ms]"
time="2021-11-29T07:25:47Z" level=info msg="photos: found 645 results for q:rupam&me public:true dist:20 year:0 month:0 quality:3 count:840 order:newest merged:true [163.221124ms]"
time="2021-11-29T07:29:03Z" level=info msg="index: updating counts"
time="2021-11-29T07:29:04Z" level=info msg="index: updating covers"
time="2021-11-29T17:55:04Z" level=info msg="cell: added place in:OcrffaVQYarN [381.961556ms]"
time="2021-11-29T17:55:07Z" level=info msg="cell: added place in:Yeiggdu71Lnz [351.575136ms]"
time="2021-11-29T17:55:08Z" level=info msg="cell: added place in:u2Df6ENBHW8o [389.196289ms]"
time="2021-11-29T17:55:22Z" level=info msg="cell: added place in:n76Y5wKgv0vs [173.383741ms]"
time="2021-11-29T17:55:22Z" level=info msg="cell: added place in:VyJAAswPGsap [351.53906ms]"
time="2021-11-29T17:55:24Z" level=info msg="cell: added place in:paaoaffCzs5q [444.685845ms]"
time="2021-11-29T17:55:29Z" level=info msg="cell: added place in:4RHNOzeWi2d9 [469.917259ms]"
time="2021-11-29T17:55:36Z" level=info msg="cell: added place in:IH4m3SnUA9Vk [233.188029ms]"
time="2021-11-29T17:55:39Z" level=info msg="cell: added place in:6Pg2sYW7z0Wf [259.471246ms]"
time="2021-11-29T17:55:54Z" level=info msg="cell: added place in:WRCQGfG8vtOm [190.161959ms]"
time="2021-11-29T17:56:08Z" level=info msg="cell: added place bd:GwrJG5dfwCOp [513.359146ms]"
time="2021-11-29T17:56:11Z" level=info msg="cell: added place in:beCiHQfEkdAa [1.291209959s]"
time="2021-11-29T17:56:11Z" level=info msg="cell: added place bt:2QEgIJydZah0 [502.719032ms]"
time="2021-11-29T17:56:22Z" level=info msg="cell: added place in:3njelHN3teXf [531.132236ms]"
time="2021-11-29T17:56:23Z" level=info msg="cell: added place in:08oQ4f2OmUCT [215.034287ms]"
time="2021-11-29T17:56:37Z" level=info msg="metadata: updated 861 photos"
time="2021-11-29T17:56:39Z" level=info msg="index: updating counts"
time="2021-11-29T17:56:40Z" level=info msg="index: updating covers"
time="2021-11-29T17:59:26Z" level=info msg="shutting down..."
time="2021-11-29T17:59:26Z" level=info msg="shutting down workers"
time="2021-11-29T17:59:26Z" level=info msg="closed database connection"
time="2021-11-29T17:59:26Z" level=info msg="http: shutting down web server"
time="2021-11-29T17:59:26Z" level=info msg="http: web server shutdown complete"
started as root
running as root
photoprism start
time="2021-11-29T18:00:08Z" level=info msg="Help us make a difference and become a sponsor today!"
time="2021-11-29T18:00:08Z" level=info msg="Visit https://docs.photoprism.org/funding/ to learn more."
time="2021-11-29T18:00:09Z" level=info msg="config: read-only mode enabled"
time="2021-11-29T18:00:09Z" level=info msg="http: enabling gzip compression"
time="2021-11-29T18:00:09Z" level=info msg="webdav: server disabled"
time="2021-11-29T18:00:09Z" level=info msg="http: starting web server at 0.0.0.0:2342"
time="2021-11-29T18:09:53Z" level=info msg="shutting down..."
time="2021-11-29T18:09:53Z" level=info msg="shutting down workers"
started as root
running as root
photoprism start
time="2021-11-29T18:10:33Z" level=info msg="Help us make a difference and become a sponsor today!"
time="2021-11-29T18:10:33Z" level=info msg="Visit https://docs.photoprism.org/funding/ to learn more."
started as root
running as root
photoprism start
time="2021-11-29T18:17:25Z" level=info msg="Help us make a difference and become a sponsor today!"
time="2021-11-29T18:17:25Z" level=info msg="Visit https://docs.photoprism.org/funding/ to learn more."
time="2021-11-29T18:17:26Z" level=info msg="config: read-only mode enabled"
time="2021-11-29T18:17:26Z" level=info msg="http: enabling gzip compression"
time="2021-11-29T18:17:26Z" level=info msg="webdav: server disabled"
time="2021-11-29T18:17:26Z" level=info msg="http: starting web server at 0.0.0.0:2342"
started as root
running as root
photoprism start
time="2021-11-29T18:22:15Z" level=info msg="Help us make a difference and become a sponsor today!"
time="2021-11-29T18:22:15Z" level=info msg="Visit https://docs.photoprism.org/funding/ to learn more."
time="2021-11-29T18:22:16Z" level=info msg="config: read-only mode enabled"
time="2021-11-29T18:22:16Z" level=info msg="http: enabling gzip compression"
time="2021-11-29T18:22:16Z" level=info msg="webdav: server disabled"
time="2021-11-29T18:22:16Z" level=info msg="http: starting web server at 0.0.0.0:2342"
started as root
running as root
photoprism start
time="2021-11-29T18:31:25Z" level=info msg="Help us make a difference and become a sponsor today!"
time="2021-11-29T18:31:25Z" level=info msg="Visit https://docs.photoprism.org/funding/ to learn more."
time="2021-11-29T19:01:13Z" level=fatal msg="dial tcp 172.18.0.3:3306: connect: connection timed out"
started as root
running as root
photoprism start
time="2021-11-29T19:01:20Z" level=info msg="Help us make a difference and become a sponsor today!"
time="2021-11-29T19:01:20Z" level=info msg="Visit https://docs.photoprism.org/funding/ to learn more."
started as root
running as root
photoprism start
time="2021-11-29T19:23:55Z" level=info msg="Help us make a difference and become a sponsor today!"
time="2021-11-29T19:23:55Z" level=info msg="Visit https://docs.photoprism.org/funding/ to learn more."
started as root
running as root
photoprism start
time="2021-11-29T19:31:06Z" level=info msg="Help us make a difference and become a sponsor today!"
time="2021-11-29T19:31:06Z" level=info msg="Visit https://docs.photoprism.org/funding/ to learn more."

```

**Comment by lastzero** [comment]
Please use GitHub Discussions for 👉 INSTALLATION PROBLEMS, SUPPORT REQUESTS, and GENERAL QUESTIONS:

  https://github.com/photoprism/photoprism/discussions

THANK YOU! 💐

Our Troubleshooting Guide helps you quickly find and fix common problems:

  https://docs.photoprism.org/getting-started/troubleshooting/

Sponsors receive direct technical support via email: https://photoprism.app/contact 📬
